dev-python/plotly/plotly-4.10.0.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# blocker dev-python/retrying
+PYTHON_COMPAT=( python3_{6..7} )
+
+inherit distutils-r1 virtualx
```

Comment:
What's virtual here for - tests? Do we need distutils_enable_tests or similar?

Reply:
Ah, a remnant from previous, will remove.